Tasks/VsTest/vstest.ts

```diff
@@ -100,30 +101,19 @@ catch (error) {
     throw error;
 }
 
-function getResolvedPattern(pattern: string): string {
-    if (path.isAbsolute(pattern)) {
-        return pattern;
-    }
-    else {
-        return path.join(systemDefaultWorkingDirectory, pattern);
+function getTestAssemblies(): string[] {
+    if (isNullOrWhitespace(searchFolder)) {
+        searchFolder = isNullOrWhitespace(sourcesDir) ? artifactsDirectory : systemDefaultWorkingDirectory;
+        tl.debug("Search directory empty, defaulting to " + searchFolder);
     }
-}
-
-function getPatternWithoutIncludeExclude(pattern: string): string {
-    return pattern.startsWith('-:') || pattern.startsWith('+:') ? pattern.substr(2) : pattern;
-}
+    tl.debug("Searching for test assemblies in: " + searchFolder);
 
-function getFilteredFilesFromPattern(pattern: string, hasQuantifier: boolean, allFiles: string[]): string[] {
-    return hasQuantifier ? getFilteredFiles(pattern, allFiles) : [pattern];
-}
-
-function getTestAssemblies(): Set<string> {
-    return new Set(ffl.findFiles(testAssembly, false, systemDefaultWorkingDirectory));
+    return tl.findMatch(searchFolder, testAssembly, null, getTaskMatchOption());
 }
 
 function getVsTestVersion(): number[] {
     let vstestLocationEscaped = vstestLocation.replace(/\\/g, "\\\\");
-    let wmicTool = tl.createToolRunner("wmic");
+    let wmicTool = tl.tool("wmic");
```

Comment:
good.  also be aware of other changes here:  https://github.com/Microsoft/vsts-task-lib/blob/master/node/docs/releases.md

specifically some folks miss argString is now arg.line.